backend/src/Models/Ballots.ts

```diff
@@ -34,20 +34,27 @@ class BallotsDB {
             status          VARCHAR,
             date_submitted  VARCHAR,
             ip_address      VARCHAR, 
-            votes           json NOT NULL
+            votes           json NOT NULL,
+            history         json
           );
         `;
         console.log(query);
         var p = this._postgresClient.query(query);
         return p.then((_: any) => {
+            return 'foo';
```

Comment:
Looks like this was some debug code that was left in?

Reply:
Lol whoops, will fix.  Great catch!